addOns/pscanrules/src/test/java/org/zaproxy/zap/extension/pscanrules/CsrfCountermeasuresScanRuleUnitTest.java

```diff
@@ -115,6 +115,28 @@ public void shouldRaiseAlertIfThereIsNoCSRFTokenFound() {
         assertEquals(alertsRaised.get(0).getEvidence(), "<form id=\"no_csrf_token\">");
     }
 
+    @Test
+    public void shouldRaiseAlertWithSortedFormFieldsInOtherInfoIfThereIsNoCSRFTokenFound() {
+        // Given
+        msg.setResponseBody(
+                "<html><head></head><body>"
+                        + "<form id=\"no_csrf_token\">"
+                        + "    <input type=\"text\" id=\"Cat\"/>"
+                        + "    <input type=\"text\" id=\"car\"/>"
+                        + "    <input type=\"text\" id=\"Bat\"/>"
+                        + "    <input type=\"text\" id=\"bar\"/>"
+                        + "    <input type=\"text\" id=\"art\"/>"
+                        + "    <input type=\"submit\"/>"
+                        + "</form></body></html>");
+        // And
+        String sortedFieldnames = "\"art\" \"bar\" \"Bat\" \"car\" \"Cat\"";
```

Comment:
This can be used directly in the assertion, or at least moved to the then section, this should not be part of the given.